modules/33acrossAnalyticsAdapter.js

```diff
@@ -235,7 +237,25 @@ function enableAnalyticsWrapper(config) {
     auctions: {},
   };
 
+  if (isGptPubadsDefined()) {
+    subscribeToGamSlots();
+  } else {
+    window.googletag = window.googletag || {};
+    window.googletag.cmd = window.googletag.cmd || [];
+    window.googletag.cmd.push(subscribeToGamSlots);
+  }
+
   analyticsAdapter.originEnableAnalytics(config);
+
+  function subscribeToGamSlots() {
+    window.googletag.pubads().addEventListener('slotRenderEnded', event => {
+      setTimeout(() => {
+        const { transactionId, auctionId } = getAdUnitMetadata(event.slot.getAdUnitPath());
+        locals.transactionManagers[auctionId] &&
```

Comment:
perhaps you could refactor this statement to something like 

```
locals.transactionManagers[auctionId]?..complete(transactionId);
```

and make it less verbose..  the same applies to this statement in the bidWon function.

Reply:
makes sense.. no need to change it